sql/src/main/java/io/crate/protocols/postgres/types/BigIntType.java

```diff
@@ -37,17 +40,20 @@
     }
 
     @Override
-    void writeTo(ByteBuffer byteBuffer, Object value) {
-        byteBuffer.putLong((long) value);
+    public int writeValue(ChannelBuffer buffer, @Nonnull Object value) {
+        buffer.writeInt(8);
+        buffer.writeLong(((long) value));
+        return 12;
```

Comment:
maybe it is better to use `INT32_BYTE_SIZE + typeLen()` just to be clear where this number comes from